core/src/main/java/com/google/errorprone/bugpatterns/UnnecessarilyFullyQualified.java

```diff
@@ -61,6 +61,7 @@
 public final class UnnecessarilyFullyQualified extends BugChecker
     implements CompilationUnitTreeMatcher {
 
+  private static final String JAVA_LANG = "java.lang";
```

Comment:
While no such package currently exist, we shouldn't also match a hypothetical `java.langSomeSuffix`:
```suggestion
  private static final String JAVA_LANG = "java.lang.";
```